### ISSUE TITLE
Unable to pass FieldProps to final form Field component

### ISSUE THREAD

**Issue by kybarg** [lookfirst/mui-rff#37]
**Describe the bug**
```react
<TextField format={val => val.trim()} name="name" label="Name" type="text" />
```
In this case **format** is `final-form` prop, but it is not passed to `Field` component

**Additional context**
Here I can see that `Field` component supports only **name** prop and [FieldProps](https://final-form.org/docs/react-final-form/types/FieldProps) are passed to **Material-UI** `TextFIeld` component
https://github.com/lookfirst/mui-rff/blob/555e55ff08b1516c929076f49871771a541b54a3/src/TextField.tsx#L15-L24

**Comment by lookfirst** [comment]
You are correct... I'll fix. Thanks for the report!

**Comment by lookfirst** [comment]
Sorry for the delay. I've made fieldProps more consistent. You can now pass in a `fieldProps` property to any of the elements to define them. You could even override `name` and `render`. Published as 1.5.0.

**Comment by lookfirst** [comment]
Missed one FieldProps in the checkbox component... 1.5.2 fixes it, but it shouldn't affect you...

**Comment by barnc** [comment]
I think KeyboardDatePickerField is also missing this.

**Comment by lookfirst** [comment]
@barnc Thanks for your comment. I believe it is in there. Can you please be more specific?

https://github.com/lookfirst/mui-rff/blob/master/src/KeyboardDatePicker.tsx#L14

**Comment by barnc** [comment]
Maybe I'm missing something but there's no way of passing props to the <Field> component there - think the field props are in the wrong place on line 23.

**Comment by lookfirst** [comment]
@barnc Oh ffs... you're totally right... thanks for your patience in explaining things. I've published 1.5.7 now that fixes this. Thank you so much!
